iac-extensions/arm/src/main/java/org/sonar/iac/arm/tree/impl/bicep/importdecl/ImportAsClause.java

```diff
@@ -37,4 +37,13 @@ public ImportAsClause(SyntaxToken keyword, Identifier alias) {
   public List<Tree> children() {
     return List.of(keyword, alias);
   }
+
```

Comment:
I'm really surprised, that this tree did not have getters so far. How this is tested? Why there is no getter for the alias? And yeah, I acknowledge, that the tree is not implemented in this PR.

Reply:
Thank you for pointing out this method. I wasn't aware of such a magic method. We should avoid testing trees that way.